### PR TITLE
[3-50] Supplement translation of all pages

### PR DESCRIPTION
**Pull request by GeranSkot**
Code refactor, translation of all available pages.
1.Adding a library for working with cookies 
2.Embedding cookies to store language settings 
3.Embedding a footer in the logging page 
4.Translation of all available pages
5.Added header in utils.service.ts (for .post)
6.Added 'void' in auth.service.ts, to navigates. (Fixed eslint error)

### CODE REVIEW

File: exalearn-ui/src/app/pages/test-page/test-page.component.ts [hunk 1]
```diff
@@ -18,6 +18,6 @@ export class TestPageComponent implements OnInit {
 		this.testQuestions = questions;
 		//TODO use user state from backend. For the test, because then we will accept the user's language from the backend.
 		//TODO When the page is reloaded, the state is cleared and we have no language
```
Comment (vstrabko) [inline exalearn-ui/src/app/pages/test-page/test-page.component.ts]: Do we still need this second "TODO"?

**Review by vstrabko — APPROVED** [review]

**Comment by vstrabko** [comment]
Nice one!

File: exalearn-ui/src/app/pages/login-page/login-page.component.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import { Component } from '@angular/core';
 import { FormControl, FormGroup, Validators } from '@angular/forms';
 import { AuthService } from '../../services/auth.service';
-
 import { UserAuth } from '../../interfaces/interfaces';
```
Comment (KotdenGit) [inline exalearn-ui/src/app/pages/login-page/login-page.component.ts]: Usually, an empty line is used to separate the imports of built-in Angular modules and application modules.

Comment (GeranSkot) [reply exalearn-ui/src/app/pages/login-page/login-page.component.ts]: Fixed, i just used fix file :c

**Review by KotdenGit — APPROVED** [review]

**Pull request merged**